backend/rds.ts

```diff
@@ -89,12 +92,39 @@ export async function ensureDatabase() {
 
 async function createDatabase() {
   await executeStatementInDatabase(null, "CREATE DATABASE " + dbName);
-  await executeStatement(
-    "CREATE TABLE Client (Id VARCHAR(255) PRIMARY KEY NOT NULL, LastMutationID BIGINT NOT NULL)"
-  );
-  await executeStatement(
-    "CREATE TABLE Shape (Id VARCHAR(255) PRIMARY KEY NOT NULL, Content TEXT)"
-  );
+
+  await executeStatement(`CREATE TABLE Cookie (
+    Version BIGINT NOT NULL)`);
+  await executeStatement(`CREATE TABLE Client (
+    Id VARCHAR(255) PRIMARY KEY NOT NULL,
+    LastMutationID BIGINT NOT NULL)`);
+  await executeStatement(`CREATE TABLE Shape (
+    Id VARCHAR(255) PRIMARY KEY NOT NULL,
+    Content TEXT NOT NULL,
+    Version BIGINT NOT NULL)`);
```

Comment:
We should probably add an index in Version. Won't matter for our use case but good to show that it's the right thing to do (so it doesn't have to scan).